force-app/main/default/classes/AccountTriggerHandler.cls

```diff
@@ -0,0 +1,50 @@
+/**
+* @description The Account Trigger Handler class
+* @author Oxana Suvorova
+*/
+public with sharing class AccountTriggerHandler {
+
+    /**
+    * @description Before Insert method.
+    * @param accounts The list of new inserted Accounts: Trigger.new
+    **/
+    public static void beforeInsert(List<Account> accounts) {
+        for (Account acc : accounts) {
+            // When an account is inserted change the account type to 'Prospect' if there is no value in the type field.
+            if (acc.Type == null) {
+                acc.Type = 'Prospect';
+            }
+            // When an account is inserted copy the shipping address to the billing address.
+            acc.BillingStreet = acc.ShippingStreet;
+            acc.BillingCity = acc.ShippingCity;
+            acc.BillingState = acc.ShippingState;
+            acc.BillingPostalCode = acc.ShippingPostalCode;
+            acc.BillingCountry = acc.ShippingCountry;
```

Comment:
Try adding some validation that only update the the billing fields if all the shipping fields are there.